configuration/items.md

```diff
@@ -242,17 +258,17 @@ FIRST_QUARTER=🌓 First Quarter
 ```
 
 ## Icons
-OpenHAB provides you a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
+OpenHAB provides a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
 However if you wish to use custom icons you need to place them inside the `conf/icons/classic/` folder.
 These icons will be used in all of the openHAB frontends.
-The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required).
+The images must be in `.png` or `.svg` format, and have a name with only small letters and a hyphen or underscore (if required).
```

Comment:
@rkoshak you might want to change "small letters" to "lowercase characters" here around line 264.